test/Microsoft.Data.Sqlite.Tests/SqliteConcurrencyTest.cs

```diff
@@ -1,233 +0,0 @@
-
-                    Assert.Equal(raw.SQLITE_LOCKED, ex.SqliteErrorCode);
-                    var message = raw.sqlite3_errmsg(connection.Handle);
-                    Assert.Equal(Strings.SqliteNativeError(raw.SQLITE_LOCKED, message), ex.Message);
-                }
-
-                dropCommand.ExecuteNonQuery();
-                Assert.Throws<SqliteException>(() => dropCommand.ExecuteNonQuery());
-            }
-        }
-
-        [Fact]
-        public void It_throws_sqlite_busy_on_deadlock()
-        {
-            using (var connection = CreateConnection())
-            {
-                var selectCommand = connection.CreateCommand();
-                connection.Open();
-                selectCommand.CommandText = "SELECT * FROM a;";
-                using (var connection2 = CreateConnection())
-                {
-                    var dropCommand = connection2.CreateCommand();
-                    dropCommand.CommandTimeout = 1;
-                    connection2.Open();
-                    dropCommand.CommandText = "DROP TABLE a;";
-                    using (var reader = selectCommand.ExecuteReader())
-                    {
-                        reader.Read();
-                        var ex = Assert.Throws<SqliteException>(() => dropCommand.ExecuteNonQuery());
-
-                        Assert.Equal(raw.SQLITE_BUSY, ex.SqliteErrorCode);
-
-                        var message = raw.sqlite3_errstr(raw.SQLITE_BUSY);
-                        Assert.Equal(Strings.SqliteNativeError(raw.SQLITE_BUSY, message), ex.Message);
-                    }
-
-                    dropCommand.ExecuteNonQuery();
-                    Assert.Throws<SqliteException>(() => dropCommand.ExecuteNonQuery());
-                }
-            }
-        }
-
-        [Fact]
-        public void Prevents_timeout_exception()
-        {
-            using (var connection1 = CreateConnection())
-            {
-                using (var connection2 = CreateConnection())
-                {
-                    connection1.Open();
-                    connection2.Open();
-
-                    var selectCommand = connection1.CreateCommand();
-                    selectCommand.CommandTimeout = 30;
-                    selectCommand.CommandText = "SELECT * FROM a;";
-
-                    var insertCommand = connection2.CreateCommand();
-                    insertCommand.CommandTimeout = 10;
-                    insertCommand.CommandText = "INSERT INTO a VALUES ( 1);";
-
-                    var startRead = new ManualResetEventSlim(false);
-
-                    var t1 = Task.Factory.StartNew(() =>
-                        {
-                            using (var reader = selectCommand.ExecuteReader())
-                            {
-                                startRead.Set();
-                                Thread.Sleep(insertCommand.CommandTimeout * 1000 / 2); // delay task 2, but not beyond its timeout
```

Comment:
lol, this line alone took 5 seconds. All the tests now pass for me in less than that.